src/routes/root.css

```diff
@@ -56,6 +56,11 @@ h4.heading {
   margin-bottom: 0;
 }
 
+p.help-text {
```

Comment:
Maybe naming this to `p.small` would make more sense instead of being too specific.